addons/mail/static/tests/tours/discuss_channel_public_tour.js

```diff
@@ -66,6 +66,47 @@ registry.category("web_tour.tours").add("discuss_channel_public_tour.js", {
             content: "Check message contains the attachment",
             trigger: '.o-mail-Message .o-mail-AttachmentCard:contains("text.txt")',
         },
+        {
+            content: "Click on add reaction",
+            trigger: ".o-mail-Message [title='Add a Reaction']",
+            run: "click",
+        },
+        {
+            content: "Emoji picker is shown",
+            trigger: ".o-EmojiPicker",
+        },
+        {
+            content: "Choose emoji",
+            trigger: ".o-Emoji:contains('🙂')",
+            run: "click",
+        },
+        {
+            content: "Reload page (fetch reactions)",
+            trigger: ".o-mail-Message",
+            run() {
+                location.reload();
+            },
```

Comment:
Ok for now, too bad we can't rely on the bus in tours yet.

Are you sure this is always reloading after the "add emoji" RPC has finished processing?

It looks like this could introduce race conditions, but I'm not sure if there is a single test cursor, order might be guaranteed.